Dockerfile

```diff
@@ -7,8 +7,8 @@ RUN git clone https://github.com/ElrondNetwork/elrond-config-devnet --branch=D1.
 RUN git clone https://github.com/ElrondNetwork/elrond-config-mainnet --branch=v1.3.42.0-rosetta1 --depth=1
```

Comment:
We don't reference a new config just yet, since the `TrieSyncerVersion = 3` isn't required by Rosetta at the moment. The config will be updated along with `v1.3.44`.